source/content/guides/decoupled/overview/04-site.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Pantheon Front-End Sites
+subtitle: Site Creation
+description: Create and configure a Front-End Site.
+tags: [webops, workflow, decoupled]
+contributors: [joa-pan, backlineint, cobypear, hckia]
+layout: guide
+showtoc: true
+permalink: docs/guides/decoupled/overview/site
+anchorid: site
+reviewed: "2023-03-16"
+contenttype: [guide]
+innav: [false]
+categories: [create]
+cms: [decoupled]
+audience: [development]
+product: [decoupled]
+integration: [--]
+---
+
+This section provides configuration information for Front-End Sites.
+
+## Prerequisites
+
+* The **Decoupled** tab is enabled in your dashboard after onboarding.
+
+* Your site repository is in GitHub.
+
+* You have a domain to use for the Front-End site.
+
+* You will use Next.js or Gatsby as the frontend framework.
+
+* (Optional). You are using the latest version of Drupal, or WordPress for the CMS.
+
+* (Optional). If using a pre-configured starter kit template, you will use one of the following combinations:
+
+    * [Drupal and Next.js](/guides/decoupled/drupal-nextjs-frontend-starters/)
+
+    * [WordPress and Next.js](/guides/decoupled/wp-nextjs-frontend-starters/)
+
+    * [WordPress and Gatsby](/guides/decoupled/wp-gatsby-frontend-starters/)
+
+    > **NOTE**: Other starter kit configurations are not currently supported.
+
+
+### Compatibility
+
+You can create an application from scratch by importing a repository or you can use the Pantheon starter kit templates to develop a Front-End Site.
+
+You can also create a site without a CMS. This is done by importing the repository and using Next.js or Gatsby as the static site generator.
```

Comment:
This is really just a nitpick so feel free to disregard :)

```suggestion
You can also create a site without a CMS. This is done by utilizing the import repository feature during site creation.
```

I don't love the faux constraint of 'Next.js or Gatsby'. I know there is some future work to be done to make the drop down menu more accurate as to the type of rendering, not the framework (at the moment next = ssr, gatsby = ssg). I'm wondering if changing this message here a bit can clear up some ambiguity? Or will this cause more confusion?

Reply:
Committed!